department-of-veterans-affairs/vets-website
# Forms-system DirectDeposit bug

## Description

`src/platform/forms-system/src/js/definitions/directDeposit.jsx` does not render on the Review page of forms correctly

The error message below is displayed when the chapter with it is opened
```
Uncaught Error: uiSchema order list contains extraneous properties 'view:directDepositInfo', 'view:bankInfoHelpText'
    at orderProperties (utils.js?8964:306)
    at ObjectField.eval (ObjectField.jsx?2f0d:36)
    at ObjectField.eval [as orderAndFilterProperties] (_createFlow.js?3414:68)
    at ObjectField.render (ObjectField.jsx?2f0d:102)
    at finishClassComponent (react-dom.development.js?61bb:17160)
    at updateClassComponent (react-dom.development.js?61bb:17110)
    at beginWork (react-dom.development.js?61bb:18620)
    at HTMLUnknownElement.callCallback (react-dom.development.js?61bb:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?61bb:237)
    at invokeGuardedCallback (react-dom.development.js?61bb:292)
```
## Testing done
- go to http://localhost:3001/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/review-and-submit
- open the accordion "Personal Information"
- notice in browser console logs the above error message appears

## Screenshots


## Acceptance criteria

## Definition of done
